lib/PropertiesPanel.js

```diff
@@ -1375,7 +1375,7 @@ function setContentEditableSelection(node, selection) {
 }
 
 function isImplicitRoot(element) {
-  return element.id === '__implicitroot';
+  return element && !!element.isImplicit;
```

Comment:
Similar to [bpmn-properties-panel](https://github.com/bpmn-io/bpmn-properties-panel/pull/102/files#r705006067) we should make this backward compatible to avoid it's breaking older bpmn-js versions.